java/src/main/java/com/powsybl/dataframe/network/adders/RatioTapChangerDataframeAdder.java

```diff
@@ -0,0 +1,72 @@
+package com.powsybl.dataframe.network.adders;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.SeriesMetadata;
+import com.powsybl.dataframe.update.UpdatingDataframe;
+import com.powsybl.iidm.network.*;
+
+import java.util.List;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ * @author Sylvain Leclerc <sylvain.leclerc@rte-france.com>
+ */
+public class RatioTapChangerDataframeAdder implements NetworkElementAdder {
+
+    private static final List<SeriesMetadata> METADATA = List.of(
```

Comment:
On load can be to true by default, and the target deadband could be to zero. For the ratio tap changer, I think that it could be great to have the possibility to set only ratios.

Reply:
See discussion synthesis about default values in comment below.